sdk/search/azure-search/src/main/java/com/azure/search/implementation/ISO8601DateDeserializer.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.search.implementation;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonTokenId;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.UntypedObjectDeserializer;
+
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Custom deserializer to deserialize strings as instances of {@link OffsetDateTime}
+ */
+final class ISO8601DateDeserializer extends UntypedObjectDeserializer {
```

Comment:
Is this class auto-generated? If not, `ISO` should be Pascal-cased to `Iso`

Reply:
fixed